pkg/protocol/sybilprotection/seatmanager/topstakers/topstakers.go

```diff
@@ -86,21 +86,45 @@ func (s *SeatManager) RotateCommittee(epoch iotago.EpochIndex, candidates accoun
 		return nil, ierrors.New("candidates must not be empty")
 	}
 
-	committee, err := s.selectNewCommittee(epoch, candidates)
+	committeeAccounts, err := s.selectNewCommitteeAccounts(epoch, candidates)
 	if err != nil {
 		return nil, ierrors.Wrap(err, "error while selecting new committee")
 	}
 
-	committeeAccounts, err := committee.Accounts()
+	// If rotating committee for epoch 0, then we can do it by default as there is no previous committee.
+	if epoch == 0 {
+		newCommittee := committeeAccounts.SeatedAccounts(committeeAccounts.IDs()...)
+
+		if err := s.committeeStore.Store(epoch, newCommittee); err != nil {
+			return nil, ierrors.Wrapf(err, "error while storing committee for epoch %d", epoch)
+		}
+
+		return newCommittee, nil
+	}
+
+	prevCommittee, err := s.committeeStore.Load(epoch - 1)
 	if err != nil {
-		return nil, ierrors.Wrapf(err, "error while getting committeeAccounts for newly selected committee for epoch %d", epoch)
+		return nil, err
+	}
+
+	// If there is no previous committee, then we can assign seats by default.
+	if prevCommittee == nil {
+		newCommittee := committeeAccounts.SeatedAccounts(committeeAccounts.IDs()...)
```

Comment:
this is kind of the same as `if epoch == 0`

Reply:
![grafik](https://github.com/iotaledger/iota-core/assets/32371094/330bcf19-3aaf-4d0d-a454-c290b69e4dbd)
just saying..... :smiling_imp: